toolbox/+otp/RHS.m

```diff
@@ -111,9 +379,20 @@
         end
         
         function opts = odeset(obj, varargin)
+            % Convert an ``RHS`` to a struct which can be used by built-in ODE solvers.
+            %
+            % Parameters
+            % ----------
+            % varargin
+            %    Additional name-value pairs which have precedence over the values in this class.
+            %
+            % Returns
+            % -------
+            % opts : struct
+            %    An options strucutre containing the subset of properties compatible with built-in ODE solvers.
+
             opts = odeset( ...
                 'Events', obj.Events, ...
-                'InitialSlope', obj.InitialSlope, ...
```

Comment:
Why did we remove this?

Reply:
That property is y0', so like y0, it belongs in the Problem not the RHS